packages/expo-doctor/src/checks/AppConfigFieldsNotSyncedToNativeProjectsCheck.ts

```diff
@@ -0,0 +1,77 @@
+import fs from 'fs';
+import path from 'path';
+
+import { DoctorCheck, DoctorCheckParams, DoctorCheckResult } from './checks.types';
+import { isFileIgnoredAsync } from '../utils/isFileIgnoredAsync';
+
+const appConfigFieldsToSyncWithNative = [
+  'ios',
+  'android',
+  'plugins',
+  'icon',
+  'scheme',
+  'userInterfaceStyle',
+  'splash',
+  'updates',
+  'orientation',
+  'backgroundColor',
+  'primaryColor',
+  'notification',
+  'androidStatusBar',
+  'androidNavigationBar',
+  'jsEngine',
+  'locales',
+];
+
+export class AppConfigFieldsNotSyncedToNativeProjectsCheck implements DoctorCheck {
+  description = 'Check for app config fields not synced to native projects';
```

Comment:
```suggestion
  description = 'Check for app config fields that may not be synced in a non-CNG project';
```
I've been struggling with how to describe this, but CNG / non-CNG might work